audio_to_spectrogram/scripts/audio_amplitude_plot.py

```diff
@@ -42,10 +46,22 @@ def start_timer(self):
         if rate == 0:
             rospy.logwarn('You cannot set 0 as the rate; change it to 10.')
             rate = 10
-        self.timer = rospy.Timer(rospy.Duration(1.0 / rate), self.timer_cb)
+
+        timer_kwargs = dict(
+            period=rospy.Duration(1.0 / rate),
+            callback=self.timer_cb,
+            oneshot=False,
+        )
+        if (LooseVersion(pkg_resources.get_distribution('rospy').version) >=
+                LooseVersion('1.12.0')):
```

Comment:
how about add checking `use_sim_time`?

Reply:
Thank you.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2764/commits/85af1afd8f2a5eb29a42ce8d38fef13a1f707ef8
I added code to check `use_sim_time`